test/e2e/operator_test.go

```diff
@@ -3579,6 +3579,47 @@ func waitForDeploymentEnvVar(t *testing.T, cl client.Client, deployment *appsv1.
 	return err
 }
 
+// waitForDeploymentCompleteWithCleanup waits for a deployment to complete its rollout, then waits for the old
+// generation's pods to finish terminating.
+func waitForDeploymentCompleteWithOldPodTermination(t *testing.T, cl client.Client, deploymentName types.NamespacedName, timeout time.Duration) error {
```

Comment:
```suggestion
// waitForDeploymentCompleteWithOldPodTermination waits for a deployment to complete its rollout, then waits for the old
// generation's pods to finish terminating.
func waitForDeploymentCompleteWithOldPodTermination(t *testing.T, cl client.Client, deploymentName types.NamespacedName, timeout time.Duration) error {
```

Reply:
Fixed in https://github.com/openshift/cluster-ingress-operator/pull/904/commits/39c6aef16c999bc4e05e948238454439e8c21b3d.